tests/integration/test_tableclasses_vectorcassandratable.py

```diff
@@ -44,8 +46,15 @@ def test_crud(self, db_session, db_keyspace):
         query_theta = 1 * math.pi * 2 / (2 * N)
         ref_vector = [math.cos(query_theta), math.sin(query_theta)]
         ann_results = list(t.ann_search(ref_vector, n=4))
-        assert {r["row_id"] for r in ann_results[:2]} == {"theta_1", "theta_0"}
-        assert {r["row_id"] for r in ann_results[2:4]} == {"theta_2", "theta_7"}
+        assert [r["row_id"] for r in ann_results] == [
+            "theta_1",
```

Comment:
Hmm, I am not sure we can assume 0<-->1 and 2<-->7 have no right to appear swapped (IIRC each vector in the two pairs has the same cos distance from query vector, no? Admittedly quite some time since I last did touch this)

Reply:
Oh, that's a set and not a list !
I wasn't awake when I touched this...

Reply:
reverted